packages/client/package.json

```diff
@@ -1,9 +1,24 @@
 {
   "name": "@penumbra-zone/client",
-  "version": "1.0.0",
-  "description": "",
+  "version": "1.0.16",
+  "description": "Package for connecting to Prax extension",
+  "type": "module",
+  "files": [
+    "dist"
+  ],
+  "exports": {
+    ".": "./dist/prax.js",
+    "./global": "./dist/global.js",
+    "./src/global": "./dist/global.js",
+    "./get-port": "./dist/get-port.js"
```

Comment:
So this is kinda the demo of how we can make external packages a bit nicer.

`"./src/global": "./dist/global.js",` <-- the way that internal packages require the exports
`"./global": "./dist/global.js",` <-- external packages get the ability to lose the `/src/` part of the import
`".": "./dist/prax.js",` <-- the default `.` is the only harmonized internal/external import that drops the need for `/src/`. In this case, gave that honor to prax.js as it's likely the most common import for this package.

Reply:
from my example

https://github.com/penumbra-zone/web/blob/b1a2a334fef98ddbcd92134eafcc47cf9a0b6f55/packages/client/package.json#L10-L28

the types package, which is private in example and needs no publication, remains unchanged

Reply:
The types package is a dependency of this package. If we do not publish it, it will throw an error when the external app goes to npm install it